backend/apps/github/api/search/user.py

```
@@ -6,7 +6,18 @@
 
 
 def get_users(query, attributes=None, limit=25, page=1, searchable_attributes=None):
-    """Return users relevant to a search query."""
+    """Return users relevant to a search query.
+
+    Args:
+        query (str): The search query string.
+        attributes (list, optional): List of attributes to retrieve. Defaults to None.
+        limit (int): Maximum number of users to return. Defaults to 25.
+        page (int): The page number for pagination. Defaults to 1.
+        searchable_attributes (list, optional): List of attributes to restrict the search to. Defaults to None.
```

Ruff (0.8.2)

Line too long (111 > 99)

(E501)